nuxeo-core/nuxeo-core/src/main/resources/OSGI-INF/coldstorage-contrib.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0"?>
+<component name="org.nuxeo.ecm.core.event.coldstorage.contribution">
+
+  <!-- define the event that will be fired by the scheduler -->
```

Comment:
Not clear, you're actually defining the scheduler itself along with the event it's firing, I would just remove the comment.